### ISSUE TITLE
1.5.0 beta.2 enable Egress also gives Multi cluster enabled in default profile

### ISSUE THREAD

**Issue by kramvan1** [istio/istio#21448]
**Bug description**

Its the default in the profile when egress is enabled
https://github.com/istio/istio/blob/1.5.0-beta.2/operator/data/profiles/default.yaml#L450-L455
and in the helm https://github.com/istio/istio/blob/1.5.0-beta.2/manifests/gateways/istio-egress/templates/preconfigured.yaml#L2

I don't think mulit cluster should be enabled in the default cluster when egress is enabled.

**Expected behavior**

**Steps to reproduce the bug**

**Version (include the output of `istioctl version --remote` and `kubectl version` and `helm version` if you used Helm)**

**How was Istio installed?**

**Environment where bug was observed (cloud vendor, OS, etc)**

**Comment by kramvan1** [comment]
cc @linsun

**Comment by sdake** [comment]
@kramvan1 This has been reported as https://github.com/istio/istio/issues/21440. This has been resolved with https://github.com/istio/istio/pull/21441.

Thank you for the report.

It appears during either our mono repo purge or the e2e testing purge, the operator test cases, the standalone operator test cases were removed. As such, we have targeted testing around the area of the standalone operator in this issue: https://github.com/istio/istio/issues/21435.

If you feel the analysis done in this closure is incorrect or require more information, feel ree to reopen the issue.

Closing for now.
